pixi.toml

```diff
@@ -1,25 +1,59 @@
-[project]
+[workspace]
 name = "aligator"
 version = "0.16.0"
 channels = ["conda-forge"]
 description = "A primal-dual augmented Lagrangian-type solver for trajectory optimization"
 platforms = ["linux-64", "osx-64", "osx-arm64"]
 license = "BSD-2-Clause"
 license-file = "LICENSE"
+readme = "README.md"
+repository = "https://github.com/Simple-Robotics/aligator"
+preview = ["pixi-build"]
 
-[build-dependencies]
+[package]
+name = { workspace = true }
+version = { workspace = true }
+
+[package.build]
+backend = { name = "pixi-build-cmake", version = "*" }
+
+[package.build.config]
+extra-args = [
+  "-DBUILD_TESTING=OFF",
+  "-DGENERATE_PYTHON_STUBS=ON",
+]
+
+[package.build-dependencies]
+cmake = ">=3.22"
 ccache = ">=4.9.1"
-cmake = ">=3.10"
 cxx-compiler = ">=1.7.0"
-ninja = ">=1.11"
 pkg-config = ">=0.29.2"
 
```

Comment:
ccache is not used by pixi-build.
cxx-compiler is managed by pixi-build-cmake.
Same with cmake.

cf https://prefix-dev.github.io/pixi-build-backends/backends/pixi-build-cmake/#required-dependencies